### PR TITLE
Added endpoints in resources

### PR DESCRIPTION
**Pull request by kacper516**

### CODE REVIEW

File: resources/models.py [hunk 1]
```diff
@@ -2,30 +2,34 @@
...
 
+
 class Department(Base):
     __tablename__ = 'Departments'
     ID_department = Column(Integer, primary_key=True)
     Department_name = Column(String)
     Description = Column(String)
     Contact_info = Column(String)
 
+
```
Comment (flexer89) [inline resources/models.py:16]: did you use any code formatter? or just add it manually

File: resources/__pycache__/models.cpython-312.pyc
Comment (flexer89) [inline resources/__pycache__/models.cpython-312.pyc:1]: add it to gitignore, we don't need it in repo

File: .gitignore
Comment (flexer89) [inline .gitignore:1]: what is hms_env

Comment (kacper516) [reply .gitignore:1]: Sorry forgot about that, it's my virtual env for make some test.

File: resources/models.py (hunk 1 above)
Comment (kacper516) [reply resources/models.py:16]: It's from PEP 8 style guide: https://peps.python.org/pep-0008/#blank-lines.
>"Surround top-level function and class definitions with two blank lines."

File: resources/__pycache__/models.cpython-312.pyc
Comment (kacper516) [reply resources/__pycache__/models.cpython-312.pyc:1]: Sure

**Review by flexer89 — APPROVED** [review]

**Pull request merged**